packages/block-library/src/navigation/index.php

```diff
@@ -1488,30 +1496,35 @@ function block_core_navigation_update_ignore_hooked_blocks_meta( $post ) {
 	return $post;
 }
 
-// Before adding our filter, we verify if it's already added in Core.
-// However, during the build process, Gutenberg automatically prefixes our functions with "gutenberg_".
-// Therefore, we concatenate the Core's function name to circumvent this prefix for our check.
+/*
+ * Before adding our filter, we verify if it's already added in Core.
+ * However, during the build process, Gutenberg automatically prefixes our functions with "gutenberg_".
+ * Therefore, we concatenate the Core's function name to circumvent this prefix for our check.
+ */
 $rest_insert_wp_navigation_core_callback = 'block_core_navigation_' . 'update_ignore_hooked_blocks_meta';
```

Comment:
@swissspidy My local PHPCS gives me a warning here. Will this also cause an Issue with Core merge?

Reply:
I disabled PHPCS for this line.

Reply:
@getdave what's the warning. Note that it's usually recommended to use `// phpcs:ignore` comments just for that particular warning, e.g. `// phpcs:ignore WordPress.DateTime.CurrentTimeTimestamp.Requested` or whatever. Makes it much clearer what the intention is.

Reply:
> String concat is not required here; use a single string instead

All I get is that

Reply:
I think the rule might be `Generic.Strings.UnnecessaryStringConcat.Found`.

Reply:
✅